Sources/LanguageServerProtocol/Requests/InlayHintsRequest.swift

```diff
@@ -0,0 +1,34 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Please add any new files to CMakeLists.txt -- don't worry if you aren't setup to do a cmake build to verify it, but usually just adding the files is enough.